python-sdk/tests/sql/operators/test_export_file.py

```diff
@@ -36,6 +36,22 @@ def make_df():
     assert df.equals(pd.DataFrame(data={"col1": [1, 2], "col2": [3, 4]}))
 
 
+def test_raise_exception_for_invalid_input_type():
```

Comment:
@pankajastro Can we add a test case to test the warning message?

Reply:
Added 62bd5b71dc75cb6fc10225e1ff9392deeceab277